addons/utm/models/utm.py

```diff
@@ -42,14 +49,57 @@ def _group_expand_stage_ids(self, stages, domain, order):
         stage_ids = stages._search([], order=order, access_rights_uid=SUPERUSER_ID)
         return stages.browse(stage_ids)
 
+    def _merge_utm_campaigns(self, keep_campaign, merge_values={}):
```

Comment:
On second thoughts, I would rename `keep_campaign` to `campaign_to_keep` everywhere.

`keep_campaign` sounds more like a boolean and it's a bit confusing.